OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -534,5 +534,16 @@ public IEnumerable<Actor> ActorsInWorld()
 		{
 			return bins.SelectMany(bin => bin.Actors.Where(actor => actor.IsInWorld));
 		}
+
+		public IEnumerable<Actor> GetActorsOnRay(CPos start, CPos end)
```

Comment:
This is essentially just a quick function to get stuff working. If there is something that should be done better, please let me know.